libs/server/Resp/Parser/RespCommand.cs

```diff
@@ -1278,6 +1284,10 @@ private RespCommand FastParseArrayCommand(ref int count, ref ReadOnlySpan<byte>
                                 {
                                     return RespCommand.RPOPLPUSH;
                                 }
+                                else if (*(ulong*)(ptr + 4) == MemoryMarshal.Read<ulong>("PEXPIREA"u8) && *(uint*)(ptr + 11) == MemoryMarshal.Read<uint>("AT\r\n"u8))
```

Comment:
nit: you're verifying 'A' twice

Reply:
It's the same for other existing 9 char long command as we don't have a datatype with 3 bytes length

Reply:
Oops I missed that! Sorry about that :)